### ISSUE TITLE
Feature Request: Filters in browser extensions.

### ISSUE THREAD

**Issue by erousr** [bitwarden/browser#412]
The Filter functionality in WebVault is wonderful. It would be great to have this in the browser extensions as well?

**Comment by kspearrin** [comment]
We have much more UI challenges is the small window of the extension. Where would we put it?

**Comment by erousr** [comment]
Thats true. I would put it above the folders, just as the WebVault is designed, but perhaps and option to collapse or hide them.

Maybe it's just me but the way I use the extension is as a "main" app. I always pop it out and size up the window when i want to admin the data and that is when i miss the filters.

I find it easier to do like this than using the WebVault.
